packages/components/package.json

```diff
@@ -42,29 +42,29 @@
     "@hashicorp/ember-flight-icons": "^3.0.2",
     "dialog-polyfill": "^0.5.6",
     "ember-a11y-refocus": "^3.0.2",
-    "ember-auto-import": "^2.6.0",
+    "ember-auto-import": "^2.6.3",
     "ember-cached-decorator-polyfill": "^0.1.4",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-sass": "^10.0.1",
     "ember-composable-helpers": "^4.5.0",
-    "ember-focus-trap": "^1.0.1",
-    "ember-keyboard": "^8.1.0",
+    "ember-focus-trap": "^1.0.2",
+    "ember-keyboard": "^8.2.0",
```

Comment:
Needs to be pinned to `8.1.0` due to [conflicting dependencies in Terraform](https://github.com/hashicorp/design-system/pull/1242).

Reply:
Question: aren't minor versions supposed to be compatible with each other? Just curious.

Reply:
In theory, yes. The trouble, from the threads that I've read, is that `ember-keyboard@8.2.0` bumps `ember-modifier` from `3.x` to `4.x`, somehow/sometimes imposing this latest version of `ember-modifier` to our consumers. I'm still checking facts around this, but that's the gist as I get it.